clt_tests/tests/test-backup-restore.rec

```diff
@@ -0,0 +1,210 @@
+––– block: ./base/init –––
+––– input –––
+sleep 1;docker run --rm -e EXTRA=1 --name manticore -v /tmp/idx:/var/lib/manticore -d manticoresoftware/manticore:current
```

Comment:
Is the `sleep 1` command really required?

Reply:
Corrected